src/domain/reports/glass-admin/entities/GLASSDataMaintenanceItem.tsx

```diff
@@ -0,0 +1,28 @@
+import { PaginatedObjects } from "../../../../types/d2-api";
+import { Id, NamedRef } from "../../../common/entities/Base";
+
+export type Status = "UPLOADED" | "VALIDATED" | "COMPLETED" | "DELETED";
+
+export type Module = "AMR" | "EGASP" | "AMRIndividual" | "EAR";
+
+export interface GLASSDataMaintenanceItem {
+    id: Id;
+    fileId: Id;
+    fileName: string;
+    module: Module;
+    orgUnit: string;
+    orgUnitName: string;
+    period: string;
+    status: Status;
+}
+
+export interface GLASSModule extends NamedRef {
+    name: Module;
+    userGroups: {
+        approveAccess?: NamedRef[];
```

Comment:
As a general rule, minimize optional properties. I guess this comes from data not being set in the data store. Then the solution is setting default values when loading from the data store. 

That said, if unsure, keep it as is.